src/dashboard/pyproject.toml

```diff
@@ -0,0 +1,63 @@
+#
+# QualiCharge package
+#
+[project]
+name = "qualicharge-dashboard"
+version = "0.0.1"
+
+# Third party packages configuration
+[tool.ruff]
+line-length = 120
+indent-width = 4
+target-version = "py312"
+# Exclude a variety of commonly ignored directories.
+exclude = [
+  "migrations",
+]
+
+[tool.ruff.lint]
+ignore = [
+    "ISC001",   # single-line-implicit-string-concatenation
```

Comment:
I wouldn't ignore this. Where is it used?

Reply:
Not currently used. 
It is a recommended rule that I had seen, but I can delete it

Reply:
✅